storage/integration_test.go

```diff
@@ -2067,7 +2102,9 @@ func testObjectsIterateAllSelectedAttrs(t *testing.T, bkt *BucketHandle, objects
 		EndOffset:   "obj2",
 	}
 	var selectedAttrs []string
-	for k := range attrToFieldMap {
+	// Iterate over proto fields instead because it doesn't have MediaLink.
+	// TODO(noahdietz): figure out how to handle this better.
+	for k := range attrToProtoFieldMap {
```

Comment:
Is this OK @tritone ? They JSON and gRPC support the same set of fields in their respective Object schema, except gRPC doesn't have a MediaLink field...

Reply:
I might just revert this given the new logic for building a fieldmask...